modules/ROOT/pages/tuning-domains.adoc

```diff
@@ -0,0 +1,17 @@
+= Domains
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Besides the listed benefits on the Shared Resources MuleSoft documentation, domains are also a performance choice when deploying multiple services on the same on-premises Mule runtime engine (Mule). By having a centralized point for all the shared resources, domains help the class-loading process, therefore metaspace memory usage, to be more efficient, especially because dependencies of the domain (those declared in the `pom.xml` file) are also shared in the domain apps.
+
+== Example using Domain
+To give an example, consider having a set of HTTP Proxy applications that also apply one API Gateway policy, such as IP whitelist. Each application interacts with the same backend.
```

Comment:
```suggestion
Imagine that you have a set of HTTP Proxy applications that also apply one API Gateway policy, such as an IP whitelist. Each application interacts with the same back end. In a Mule instance configured to have 2 GB of heap memory and 256 MB of metaspace, deploying those applications independently results in a noticeable reduction in TPS (transactions per second) if you deploy 60 or more applications. But if you deploy these applications in a domain that shares the backend server configuration, you can not only recover TPS performance but also do so when deploying over 100 applications.
```

Reply:
...or end after "the same back end" and use the copyedits provided just in case below....